Dockerfile

```diff
@@ -31,6 +31,8 @@ ARG NOVNC_REPO=https://github.com/novnc/novnc
 ARG NOVNC_REF=v1.0.0
 ARG SPICE_REPO=https://gitlab.freedesktop.org/spice/spice-html5.git
 ARG SPICE_REF=spice-html5-0.1.6
+ARG MKS_REPO=https://github.com/rgerganov/noVNC.git
+ARG MKS_REF=master
```

Comment:
Do we really want master? Why not pin to a commit ID?

Reply:
TIL: I can't git-clone to a commit. :/

But I'd still do
```
git clone -b master ...
git checkout $MKS_COMMIT
```

Reply:
There probably won't be any change ever to this project, but from security side, you're right.

Reply:
we cannot do a `--depth 1` then and thus increase the image size without need. any idea how to clean that up?

Reply:
we could use `git clone --shallow-since=2016-08-21` but I'm not sure which git version is used during build. 2.11 (released Nov 2016) is the git version that introduced the flag.

Reply:
This still keeps a lot of stuff if there's something newer. I think in favor of smaller images, I would go with using `master` after all. It's also what the other kids in the same file do, so it would be consistent.

If your concern is about safety i.e. that user pushing a malicious commit, we could fork the repo and only update master manually.

Reply:
> This still keeps a lot of stuff if there's something newer.

Is that so? If there's something newer, it would just have the newer commits, or not? Does that qualify as "a lot of stuff"?

Re discussion-effort-to-technical-merit ratio: it's becoming less than 1.0 about now for me, so if we don't agree -- let's just shallow-clone master and be done with it.